MonitorizareVot/Onboarding/Views/OnboardingView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  VotingStationOnboardingView.swift
+//  MonitorizareVot
+//
+//  Created by Geart Otten on 14/10/2019.
+//  Copyright © 2019 Code4Ro. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class OnboardingView: View {
+    
+    // MARK: Private variables
+    
+    private lazy var contentView: UIView = {
```

Comment:
Most UI customization should be done in Nibs instead of code, please implement it as a nib. You can create a xib with the same name and make the root view be the custom class, then load the xib in code and the first object would be the actual custom class object.